appassembler-booter/src/main/java/org/codehaus/mojo/appassembler/booter/AppassemblerBooter.java

```diff
@@ -121,6 +121,9 @@ public static URLClassLoader createClassLoader( File repoDir )
     {
         List<URL> classpathUrls = new ArrayList<URL>();
 
+        StringBuilder appClassPath = new StringBuilder();
+        boolean firstAppClassPathElement = true;
+
```

Comment:
Maybe I misunderstand a thing here but it looks like a constant which can't be changed? If so why is it there?

Reply:
Sorry I am not sure that I follow you, which variable to you believe is "constant" and misplaced?

Reply:
Sorry it looks like I misunderstood the intention.. What I don't understand is that the first ClasspathElement should not be added?

Reply:
It is the separator between elements that is only added after the first entry.